accounting/transferwise/admin.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2017-2018 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+Admin for the TransferWise app.
+"""
+
+from django.contrib import admin
+
+from accounting.common.admin import UuidModelAdmin
+from accounting.transferwise import models
+
+
+@admin.register(models.TransferWiseBulkPayment)
+class TransferWiseBulkPaymentAdmin(UuidModelAdmin):
+    """ Admin configuration for the `TransferWiseBulkPayment` model. """
+    list_display = ('date', 'sender', 'csv_path',)
+    search_fields = ('sender__user__username',)
+    readonly_fields = ('csv_path',)
```

Comment:
Could you make links like this clickable? https://stackoverflow.com/questions/1949248/how-to-add-clickable-links-to-a-field-in-django-admin seem

Reply:
No worries, I absolutely plan on doing this in https://tasks.opencraft.com/browse/OC-3637, for next week. Since it'll be a relatively simple PR, we can get it ready before the next billing cycle.

Reply:
@UmanShahzad OK - could you add it explicitly to the scope of OC-3637 then, to not forget?

Reply:
@antoviaque Done :+1: